utils.pwsh/Setup-Target.ps1

```diff
@@ -39,8 +39,8 @@ function Setup-BuildParameters {
     $VisualStudioData = Find-VisualStudio
 
     $VisualStudioId = "Visual Studio {0} {1}" -f @(
-        ([System.Version] $VisualStudioData.Version).Major
-        ([regex]::Match($VisualStudioData.Name, "\b\d{4}\b"))
+        $VisualStudioData.InstallationVersion.Major
+        ( $VisualStudioData.DisplayName -split ' ')[3]
```

Comment:
This will need adjusted to not overwrite the change to use regex.

Reply:
That change is intended - the old variant was based on the output of the CMI provider, the update uses `VSSetup` and its output instead.

@Absolucy would you be able to check what the `DisplayName` given by `VSSetup` is in your setup?

Reply:
There's no need. I checked this already before merging #178.
```
PS C:\> Get-VSSetupInstance -Prerelease:$false

InstanceId          : 142e8861
DisplayName         : Visual Studio Community 2022
InstallationVersion : 17.6.33723.286
InstallationPath    : C:\Program Files\Microsoft Visual Studio\2022\Community
InstallDate         : 11/1/2022 1:38:25 PM

InstanceId          : a63227f3
DisplayName         : Visual Studio Build Tools 2022
InstallationVersion : 17.6.33723.286
InstallationPath    : C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools
InstallDate         : 5/30/2023 1:17:32 PM
```